backend/ddueruemweb/settings.py

```diff
@@ -35,6 +35,7 @@
     'django.contrib.sessions',
```

Comment:
This is not part of this review because it is part of #29

Reply:
ok